src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/UpdateThirdPartyIdpDialog.razor.cs

```diff
@@ -53,17 +53,17 @@ public async Task GetThirdPartyIdpDetailAsync()
 
     public async Task UpdateThirdPartyIdpAsync(EditContext context)
     {
-        if (string.IsNullOrEmpty(ThirdPartyIdp.Icon))
-        {
-            OpenErrorMessage(T("Please upload ThirdPartyIdp icon"));
-            return;
-        }
+        //if (string.IsNullOrEmpty(ThirdPartyIdp.Icon))
```

Comment:
Why commit it?

Reply:
Mr. Xie said the third-party platform icon upload is not required